zmon-controller-ui/js/controllers/AlertDetailsCtrl.js

```diff
@@ -23,6 +23,9 @@ angular.module('zmon2App').controller('AlertDetailsCtrl', ['$scope', '$location'
         $scope.showActiveAlerts = true; // 1st tab, 1st button
         $scope.showAlertsInDowntime = false; // 1st tab, 2nd button
         $scope.showCheckResults = false; // 1st tab, 3rd button
+        $scope.alertDetailsSearch = { // entity filter
+            str: localStorageService.get('alertDetailsSearchStr') || ''
```

Comment:
Why do we need a child property? Can't `$scope.alertDetailsSearch` be a string itself?